pkg/store/clear.go

```diff
@@ -8,6 +8,8 @@ import (
 	"gorm.io/gorm"
 )
 
+// Clear deletes everything from the database. Useful for development.
+// Be careful in production...
 func Clear(ctx context.Context, db *gorm.DB) error {
 
```

Comment:
should we add a check for the environment here?

Reply:
or create a command line script for it instead of having that in the code

Reply:
Yeah, i'll add a `CORE_VERY_DANGEROUSLY_ALLOW_CLEAR_DB` env var, needed to run this

Reply:
I'd be quite nervous leaving this code in when we deploy a production version. A misconfigured environment could wipe the database with no warning.